runner/runners/README.md

```diff
@@ -0,0 +1,29 @@
+# Runner Impl Design #
+This package contains implementations of Runner.
+
+There are three main components of our Runner Impl:
+* Statuses
+* Controller
+* Invoker
+
+## Statuses (statuses.go) ##
+Statuses is a database of ProcessStatus. It has two facets:
+* Read Statuses (by implementing runner.Statuser). Allows reading the current state and waiting for new results.
+* Write Statuses. What writes Statuses?
+
+## Controller (single.go and queue.go) ##
+Controller offers Run() and Abort(). It offers mutual exclusion (don't run two at a time) and a queue (in queue.go; single.go doesn't). It writes to a Statuses.
+
+To actually start a process, it uses an Invoker.
+
+## Invoker (invoke.go) ##
+Invoker doesn't implement any interface; it's just a struct with useful methods. Invoker's Run method lowers a Scoot command to an Execer command. This involves:
```

Comment:
Add a sentence about what Invoker does first before describing implementations.  Struct with useful methods tells me nothing.  Its a struct that with methods for (... list things methods do...)

What does lower mean in this context? Sends a command to the Execer?